extension/src/repository/data/index.ts

```diff
@@ -91,8 +90,7 @@ export class RepositoryData extends DeferredDisposable {
     const dataStatus =
       await this.internalCommands.executeCommand<DataStatusOutput>(
         AvailableCommands.DATA_STATUS,
-        this.dvcRoot,
-        Flag.UNCHANGED
```

Comment:
[F] This was here because I was going to try and call for a full or partial update based on the path that fired the watcher. However, with the new data, it will require a more extensive rewrite to make this more efficient. Luckily the test caught the oversight.